docs/configuration.md

```diff
@@ -0,0 +1,115 @@
+# Configuring and Tuning Cluster Logging
+
+OKD cluster logging is configurable by modifying the CustomResource deployed
+in the *openshift-logging* namespace.  The [sample](../hack/cr.yaml) resource
+defines a complete cluster logging deployment which includes all the subcomponents
+of the logging stack to collect, store and visualize logs.  The `cluster-logging-operator`
+watches the CustomResources and adjusts the logging deployment accordingly.
+
+## Management State
+Cluster logging is `Managed` by the `cluster-logging-operator`.  It can be placed
+into an unmanaged state allowing an administrator to assume full control of individual
+component configurations and upgrades by modifiying the `managementState` to `Unmanaged`.
+**Note:** An `Unmanaged` deployment will not receive updates until the operator is placed back into
+`Managed` state.
+```
+  spec:
+    managementState: "Managed"
+```
+
+## Component Images
+The image for each subcomponent can be overriden by modifying the appropriate
+environment variable in the `cluster-logging-operator` Deployment.  The variables
+define the full pull spec for the image:
+
+* `CURATOR_IMAGE`
+* `ELASTICSEARCH_IMAGE`
+* `FLUENTD_IMAGE`
+* `KIBANA_IMAGE`
+* `OAUTH_PROXY_IMAGE`
+* `RSYSLOG_IMAGE`
+
+**Note:** These changes may require the `cluster-version-operator` to be scaled to `0` so
+changes to the `cluster-logging-operator deployment are not reverted.
+
+## Common Configurations
+### Memory and CPU
+Each component specification allows for adjustments to both the CPU and 
+memory limits.  This can be added to the CustomResource by defining a `resources`
+block:
+```
+  spec:
+    logStore:
+      type: "elasticsearch"
+      elasticsearch:
+        resources:
+          limits:
+            cpu:
+            memory:
+          requests:
+            cpu:
+            memory:
+```
+
+## Data Aggregation and Storage
```

Comment:
I recommend we remove storage option as outlined here: https://github.com/openshift/cluster-logging-operator/issues/63#issuecomment-448374769